app/views/partials/_post_table.html.erb

```diff
@@ -76,17 +62,19 @@
 											Cancelled on <%= p.date_cancelled.to_s %>
 
 								<% end %>
+
 								<% if p.claimer_id %>
-									<p class="mobile-only table-heading">Claimer</p>
 									<% if logged_in? and p.claimer_id == current_user.id %>
-										<%if p.date_completed.nil?%><%= link_to 'Unclaim Request', unclaim_request_path(id: p.id),data: {confirm: "Are you sure you want to cancel this request?"}%>
+										<%if p.date_completed.nil?%><%= link_to 'Unclaim Request', unclaim_request_path(id: p.id),data: {confirm: "Are you sure you want to unclaim this request?"}%>
 										<%end%>
 									<% end %>
 								<% elsif logged_in? and p.date_cancelled.nil? %>
 									<%= link_to 'Claim Request', claim_request_path(id: p.id) %>
 								<% end %>
+
+
 							</td>
-							<td>
+							<td class = "detail_toggle">
```

Comment:
New feature:
details won't be shown until the users hit view_details for mobile view